app/controllers/feedbacks_controller.rb

```diff
@@ -0,0 +1,15 @@
+require "uri"
+require "net/http"
+
+class FeedbacksController < ApplicationController
+  def create
```

Comment:
Aaa, preco to neposleme rovno z JS ako to mame v ekosysteme?

Reply:
Hadzalo mi to CORS errory + ten 'Useful' feedback nejde z formularu, ale z <a>. Takto sa nemusime zapodievat CSSkami a hackovat okolo CORS.

Reply:
To je divne kedze na ekosysteme tak normalne fungujeme. Inak useful uz GA predtym mal, tam si nieco menil?

Reply:
GA: Nie, len som to presunul a bude to posielat Useful/Not useful/Bug report namiesto Yes/No

Na ekosysteme ale mame form, nie `<a>`. Tu mame aj `<a>` aj `<form>`. Mozem do toho este vrtat, ale takto je to podla mna cistejsie.